src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseHybridFVM.cpp

```diff
@@ -302,6 +303,23 @@ void SinglePhaseHybridFVM::assembleFluxTerms( real64 const GEOSX_UNUSED_PARAM( t
   } );
 }
 
+void SinglePhaseHybridFVM::assemblePoroelasticFluxTerms( real64 const GEOSX_UNUSED_PARAM( time_n ),
+                                                         real64 const dt,
+                                                         DomainPartition & domain,
+                                                         DofManager const & dofManager,
+                                                         CRSMatrixView< real64, globalIndex const > const & localMatrix,
+                                                         arrayView1d< real64 > const & localRhs )
+{
+  GEOSX_UNUSED_VAR( dt );
+  GEOSX_UNUSED_VAR( domain );
+  GEOSX_UNUSED_VAR( dofManager );
+  GEOSX_UNUSED_VAR( localMatrix );
+  GEOSX_UNUSED_VAR( localRhs );
+
+  GEOSX_ERROR("Not yet implemented");
+
+}
+
```

Comment:
If you want to leave these functions not implemented in this PR, feel free, I will be glad to implement them in a follow-up PR. From what I have seen in your PR, nothing will prevent the new constitutive model for permeability from being used in the hybrid flow solvers.

Reply:
Yes, I won't implement this. I just want to make create all the interfaces to make sure that the design works well. You can have fun with this later on.